apis/keda/v1alpha1/scaledobject_webhook_test.go

```diff
@@ -153,9 +153,8 @@ var _ = It("shouldn't validate the so creation when the replica counts are wrong
 	err := k8sClient.Create(context.Background(), namespace)
 	Expect(err).ToNot(HaveOccurred())
 
-	Eventually(func() error {
-		return k8sClient.Create(context.Background(), so)
-	}).Should(HaveOccurred())
+	err = k8sClient.Create(context.Background(), so)
+	Expect(err).To(HaveOccurred())
```

Comment:
I've removed the use of `Eventually` in these tests according to https://github.com/kedacore/keda/pull/5962#issuecomment-2227553454.

I also set up https://github.com/vzdai/keda/pull/1 to remove `Eventually` from all other tests where it's not needed. That branch is based off of this branch (since without the changes here, some tests would fail), and once this PR gets merged, I can update the base of that PR to `kedacore/keda/main`.